test/spmd/tensor/test_tensor_ops.py

```diff
@@ -152,6 +154,82 @@ def test_zeros_like(self):
         zeros_expected = torch.zeros(4, 8)
         self.assertEqual(zeros_expected, zeros_like_dt.to_local())
 
+    @with_comms
+    def test_softmax_fwd(self):
```

Comment:
Ideally, we can merge fwd and bwd into one test right?

Reply:
Yes. We decide to temporarily keep fwd test because the bwd test fails on CPU on some cases and we will disable it on CPU with @skip_if_lt_x_gpu. Once the test passes on CPU, I'll remove fwd test.